deploy/90_rh-ssh.SSHD.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cloudingress.managed.openshift.io/v1alpha1
+kind: SSHD
+metadata:
+  name: rh-ssh
+  namespace: "${{SSHD_NAMESPACE}}"
+spec:
+  dnsName: rh-ssh
+  allowedCIDRBlocks: "${{ALLOWED_CIDR_BLOCKS}}"
+  image: "${{SSHD_IMAGE}}"
```

Comment:
same here with double curly...

Reply:
I defer to @cblecker and @mwoodson's expertise on this point.  No strong opinion from me.

Reply:
I think it makes a lot of sense to at least deploy the namespace through MCC, since it will need a service load balancer quota exemption.  Opened https://github.com/openshift/managed-cluster-config/pull/410 for this.